ui/index.d.ts

```diff
@@ -983,39 +984,30 @@ export interface UIExtensionVariables {
     reporter: ITelemetryReporter;
 }
 
-export interface IAddUserAgent {
-    addUserAgentInfo(additionalUserAgentInfo: any): void;
-}
-
 /**
  * Retrieves a user agent string specific to the VS Code extension, of the form `${extensionName}/${extensionVersion}`,
  * and appends it to the given user agent string, if it isn't already in the string. Passing in no existingUserAgent
  * will return just the extension portion to use in a user agent.
  */
 export declare function appendExtensionUserAgent(existingUserAgent?: string): string;
 
-/**
- * Adds the extension user agent to the given ServiceClient or other object support AddUserAgentInfo
- */
-export declare function addExtensionUserAgent(client: IAddUserAgent): void;
```

Comment:
In the old API, we would create a management client, then add user agent info. 
In the new API, we can provide a method that adds user agent info, which is then invoked during client construction. I don't see method tied to any management client that allows us to add user agent info post-construction. 


As an aside, the method `addUserAgentInfo` has become `addUserAgentHeader`, but I don't think it can be invoked by the client directly.